events/Random Events.txt

```diff
@@ -24,15 +24,14 @@ country_event = {
 	option = {
 		name = "Of Course!"
 		
-		treasury = -10000
+		treasury = -1000
```

Comment:
Use `add_tax_relative_income` instead of a hardcoded amount

Reply:
This could send a nation into debt though, while the event before this change makes sure you would have the 10000 needed. Is there a way to check the treasure is higher than the add_tax_relative_income amount?

Reply:
Yeah, it's used in the "Famine strikes state" and it makes paying off debt difficult, which is highly detrimental if your Secondary or Great Power status comes from prestige. It can also make AI default because it doesn't know to borrow before choosing the event option.

Reply:
I meant if there is a check for the trigger to ensure a nation has enough money, if the add_tax_relative_income is used. Searching for relative_income only shows the add_tax_relative_income results and i don't know if it just is not used anywhere or does not exist at all.

Reply:
Afaik there isn't one. For a player we definite want to use `add_tax_relative_income`, but for the ai it would be better to have a hardcoded value. Let's stick with 10k for the ai -- we can use if emulation to handle the two different values.

Alternatively, I have been planning on implementing the "pay or owe" system that many other mods use, where it effectively puts the country in debt by 10k if they don't have the cash on hand (thereby preventing bankruptcy via flavor). In this case, keep the 10k and it'll be updated to not cause bankruptcy when that system is implemented

Reply:
The event is on the yearly pulse and for smaller nations it takes a while to get 10k and having an event that can take it away each year seems quite bad.

Reply:
That's why the other option exists. No tag is forced to take the first option -- it's generally going to be the better option, but you can choose take the option that benefits the rich